postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -403,7 +408,7 @@ def _get_extra_filters_and_params(self, filter_stale_idle_conn=False):
         if self._config.dbstrict:
             extra_filters = " AND datname = %s"
             params = params + (self._config.dbname,)
-        else:
+        elif len(self._config.ignore_databases) > 0:
```

Comment:
It's great to fix these small changes like this, but I highly recommend separating them into discrete pull requests with separate purposes. The reason is that we might need to revert some functionality or block on merging changes for some reason. It's best to have granular control over changes that ship together. Additionally, release notes are tracked per-PR so this would be a fix.

Reply:
ok, will take this out and move to another PR!

Reply:
Oh ok understood. Weird because I would expect it to be unrelated, but yeah disregard my comment then.

Reply:
@edengorevoy uncovered a hacky test 💯